Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContextPrivate.h

```diff
@@ -29,6 +29,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface _WKWebExtensionContext ()
 
+/*!
+ @abstract Enables extra `browser.test` JavaScript APIs for unit testing.
+ @discussion Defaults to `YES` in debug builds.
+ */
+@property (nonatomic, getter=_inTestingMode, setter=_setTestingMode:) BOOL _testingMode;
```

Comment:
I understand that this is only for testing. But I don’t understand why it is private.

Reply:
We don't plan to expose this as part of the new Web Extension API. `_WKWebExtensionContext` will eventually become `WKWebExtensionContext` after API review. 🤞